config/locales/es.yml

```diff
@@ -3162,8 +3163,8 @@ es:
         info: "¡Sabemos que los impuestos pueden ser difíciles y estamos aquí para ayudarlo en cada paso del camino!"
         partner_info_html: "%{partner_name} prepara declaraciones de impuestos de <strong>%{zip} (%{zip_name}).</strong>"
         returning_client:
-          partner_info: Nuestro equipo en %{partner_name} está aquí para ayudarlo a presentar su solicitud nuevamente.
-          title: Bienvenida de nuevo %{name}!
+          partner_info: 'De nuevo, nuestro equipo en %{partner_name}está aquí para ayudarle a presentar su declaración . '
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.

Additionally there is a space before the `.` (period). This will look bad.